ci/build-project.ps1

```diff
@@ -10,12 +10,13 @@ param(
     [string] $build_state,
     [string] $build_target
 )
+. "$PSScriptRoot\common.ps1"
 
 # Clone the testing project
 Write-Output "Downloading the testing project from: $($test_repo_url), branch: $($test_repo_branch)"
 git clone -b "$test_repo_branch" "$test_repo_url" "$test_repo_path" --depth 1
 if (-Not $?) {
-    Throw "Failed to clone testing project from $test_repo_url."
+    Throw "Failed to clone testing project from branch: $($test_repo_branch), url: $($test_repo_url), path: $($test_repo_path)."
```

Comment:
```suggestion
    Throw "Failed to clone testing project from branch: ${test_repo_branch}, url: ${test_repo_url}, path: ${test_repo_path}."
```
is probably easier, but it works either way

Reply:
Ohh no idea this syntax existed good idea !